qt/DetailsDialog.cc

```diff
@@ -849,6 +893,34 @@ void DetailsDialog::refreshUI()
 
     ui_.privacyValueLabel->setText(string);
 
+    // myLabelsTextEdit
+    string = none;
+
+    if (ui_.labelsTextEdit->toPlainText() == QStringLiteral("Initializing..."))
```

Comment:
It looks like this safeguard is here to prevent `refreshUI()` from updating the edit field if a user is in the middle of editing it.

Maybe we could remove `"Initializing..."` altogether and test for `if (!ui_.labelsTextEdit->hasFocus())` instead?

Reply:
Using focus here has the problem that changes the user has made are overridden as soon as they click somewhere else. I think the refresh-only-once solution works best here.